src/mongoose_service.erl

```diff
@@ -71,14 +72,14 @@ stop_service(Service) ->
 
 -spec ensure_loaded(service()) -> ok.
 ensure_loaded(Service) ->
-    case is_loaded(Service) of
```

Comment:
Why the `is_loaded` check is missing now?

Reply:
Because that same `is_loaded` check is done again in `start_service`, so the only thing that `ensure_loaded` does is to `start_service` and ignore the `already_started` error.